R/legend.R

```diff
@@ -46,8 +46,8 @@ easy_remove_legend <- function(..., teach = FALSE) {
   } else {
     inputs <- lapply(vars, function(x) FALSE)
     names(inputs) <- vars
-    
-    if (teach){
+
+    if (teach) {
```

Comment:
Please be careful not to modify (even trivially) unrelated files, simply because it clutters the diff.